content/providers/01-ai-sdk-providers/10-google-generative-ai.mdx

```diff
@@ -266,6 +266,7 @@ const result = await generateObject({
 
 | Model                     | Image Input         | Object Generation   | Tool Usage          | Tool Streaming      |
 | ------------------------- | ------------------- | ------------------- | ------------------- | ------------------- |
+| `gemini-2.0-flash-exp`    | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
```

Comment:
we need this in the 2 overview pages as well (where models from diff providers are listed)

Reply:
Done.